dockers/docker-database/database_config.json

```diff
@@ -0,0 +1,24 @@
+{
+    "DISABLE_DATABASE": {
+        "redis":{
+            "port": 6379,
+            "socket": "/var/run/redis/redis.sock",
+            "databases" : ["CONFIG_DB","STATE_DB","LOGLEVEL_DB","SYSMON_DB"]
```

Comment:
>CONFIG_DB [](start = 28, length = 9)

How do you specify the database number for CONFIG_DB, etc.? Considering warm-reboot into a new image with a new database_config.json for splitting migration, we need to keep CONFIG_DB  at the same database number.

Reply:
Offline discussed, the database number doesn't change at all.

---
In reply to: [311777419](https://github.com/Azure/sonic-buildimage/pull/2182#discussion_r311777419) [](ancestors = 311777419)